### ISSUE TITLE
infinite loop for fixed_indegree, fixed_outdegree

### ISSUE THREAD

**Issue by jakobj** [nest/nest-simulator#227]
when executing the following commands
```
/iaf_psc_delta Create
[1] [1] << /rule /fixed_indegree /indegree 1 /autapses false /multapses false >> Connect
```
(or equivalently with `fixed_outdegree`) NEST hangs in an infinite loop. this most likely is the same  behaviour as was reported in trac ticket 451 and consequently fixed (see `testsuite/regressiontests/ticket-451.sli`). this is an issue if the connect routine can not finish due to the user-given constraints. this should probably be fixed for the NewConnect framework.

**Comment by jakobj** [comment]
this is also addressed with #221: a warning informs the user of a possible endless loop. as soon as that PR is merged, this issue can be closed.

**Comment by mschmidt87** [comment]
So @jakobj , can you please close this issue now?
